config/locales/models/en.yml

```diff
@@ -155,6 +155,15 @@ en:
               not_work: Only a link to a work can be listed as an inspiration.
               protected: You can't use the related works function to cite works by the protected user %{login}.
           format: "%{message}"
+        skin:
+          attributes:
+            base:
+              archive_in_title: You can't use the word 'Archive' in your skin title, sorry! (We have to reserve it for official skins.)
+              format: "%{message}"
```

Comment:
This is not needed when the error is added onto `:base`. Base means the overall model, so there is no attribute that gets added to the error message: https://guides.rubyonrails.org/active_record_validations.html#errors-base